falkordb-cluster/cluster-entrypoint.sh

```
@@ -75,6 +75,94 @@
+  echo "Taking extra measures"
+  info=$(redis-cli $AUTH_CONNECTION_STRING $TLS_CONNECTION_STRING info replication)
+  if [[ "$info" =~ role:slave ]];then
+    ip=$(echo "$info" | grep master_host | cut -d':' -f2)
+  elif [[ "$info" =~ role:master ]];then
+    ip=$(echo "$info" | grep ip | cut -d'=' -f2 | cut -d',' -f1)
+  fi
+
+  for i in $ip;do
+    ans=$(grep $i $DATA_DIR/nodes.conf)
+    if [[ -z $ans ]];then
+      echo "The result is empty: $result"
```

Shellcheck (0.10.0)

[warning] result is referenced but not assigned.

(SC2154)